aiqc/__init__.py

```diff
@@ -985,7 +1008,28 @@ def to_pandas(
 			columns:list = None, 
 			samples:list = None
 		):
-			return Dataset.Tabular.to_pandas(id, columns, samples)
+			df = Dataset.Tabular.to_pandas(id, columns, samples)
+			word_counts, feature_names = Dataset.Text.get_feature_matrix(df)
+			df = pd.DataFrame(word_counts.todense(), columns = feature_names)
+			return df
+
+		
+		def to_numpy(
+			id:int, 
+			columns:list = None, 
+			samples:list = None
+		):
+			df = Dataset.Tabular.to_pandas(id, columns, samples)
+			word_counts, feature_names = Dataset.Text.get_feature_matrix(df)
+			return word_counts, feature_names
+
+
+		def get_feature_matrix(
+			dataframe:object
+		):
+			count_vect = CountVectorizer()
+			word_counts = count_vect.fit_transform(dataframe[Dataset.Text.column_name].tolist())
+			return word_counts, count_vect.get_feature_names()
```

Comment:
Keeping in mind for longterm, this metadata may be good for a Text table. We'll see.

Reply:
okay, this is just boiler plate code to get started, planning on adding support for other transformation techniques as well, plus refactoring any meta data into tables as use cases show up in future